cmd/jujud/agent/machine.go

```diff
@@ -849,7 +852,7 @@ func (a *MachineAgent) updateSupportedContainers(
 		return errors.Annotatef(err, "cannot load machine %s from state", tag)
 	}
 	if len(result) != 1 {
-		return errors.Annotatef(err, "expected 1 result, got %d", len(result))
+		return errors.Errorf("expected 1 result, got %d", len(result))
```

Comment:
Drive by, this will return nil, if this ever happens, because we guard against `err` above.